CMIP6_experiment_id.json

```diff
@@ -4425,6 +4425,35 @@
             ],
             "tier":"1"
         },
+        "historical-cmip5":{
+            "activity_id":[
+                "CMIP"
+            ],
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"CMIP5 historical experiment, using CMIP5-era forcing that concluded in 2005",
+            "end_year":"2005",
+            "experiment":"all-forcing simulation of the recent past (CMIP5-era forcing)",
+            "experiment_id":"historical-cmip5",
+            "min_number_yrs_per_sim":"156",
+            "parent_activity_id":[
+                "CMIP"
+            ],
+            "parent_experiment_id":[
+                "piControl-cmip5"
```

Comment:
Should the parent be "piControl-cmip5-spinup"? In which case you will probably need an experiment id entry for  "piControl-cmip5-spinup" for which the parent would be "none".